templates/cloud/index.html

```diff
@@ -39,6 +36,7 @@
       <h1>云计算的标准操作系统</h1>
       <p class="p-heading--four">70％的公有云工作负载和54％的OpenStack云<a href="#fn1" id="r1">*</a></p>
       <p><a href="https://www.ubuntu.com/download/cloud" class="p-button--positive">安装OpenStack</a></p>
+      <p>如您已明确企业的需求，可直接<a href="https://pages.ubuntu.com/contact-us-in-chinese.html" class=""p-button--neutral is-inline">联系我们</a>获得技术支持以及认证服务。</p>
```

Comment:
You are closing the class tag before the class names: `class="p-button--neutral is-inline"`

Reply:
Maybe just leave it a link as the inline button doesn't have padding on the right